Formula/vvenc.rb

```diff
@@ -0,0 +1,31 @@
+class Vvenc < Formula
+  desc "Fraunhofer Versatile Video Encoder"
+  homepage "https://github.com/fraunhoferhhi/vvenc"
+  url "https://github.com/fraunhoferhhi/vvenc/archive/refs/tags/v1.8.0.tar.gz"
+  sha256 "119970f1e00667045eb12775db10611fc04f9158348144913c9e233f98664714"
+  license "BSD-3-Clause-Clear"
+  head "https://github.com/fraunhoferhhi/vvenc.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "ffmpeg" => :test
+
+  resource("test_video") do
+    url "https://github.com/pthariensflame/homebrew-assorted/raw/master/test-fixtures/test.hevc"
+    sha256 "1fe82788190d1045ae42aaa3b586032ec306b5aaf493b369d115fe7456497b80"
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DVVENC_INSTALL_FULLFEATURE_APP=1"
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    resource("test_video").stage testpath
+    system Formula["ffmpeg"].bin/"ffmpeg", "-i", testpath/"test.hevc", "-c:v", "rawvideo",
+           "-pix_fmt", "yuv420p", testpath/"test.yuv"
```

Comment:
Maybe use a `.yuv` resource so that we can skip the `ffmpeg` part of this test? It might help this finish within the time limit.